make/modules/jdk.sctp/Java.gmk

```diff
@@ -47,10 +47,24 @@ ifeq ($(call isTargetOs, aix), true)
   EXCLUDE_FILES += $(SCTP_IMPL_CLASSES)
 endif
 
+ifeq ($(OPENJDK_TARGET_OS_ENV), bsd.openbsd)
+  EXCLUDE_FILES += $(SCTP_IMPL_CLASSES)
+  ## WORKAROUND need to provide Impl classes that throw UnsupportedOperationException
+  SRC_SUBDIRS += macosx/classes
+endif
+
+ifeq ($(OPENJDK_TARGET_OS_ENV), bsd.netbsd)
+  EXCLUDE_FILES += $(SCTP_IMPL_CLASSES)
+  ## WORKAROUND need to provide Impl classes that throw UnsupportedOperationException
+  SRC_SUBDIRS += macosx/classes
+endif
+
 ifeq ($(call isTargetOsType, unix), true)
   ifeq ($(call isTargetOs, macosx aix), false)
```

Comment:
This too can be remade to this:

```suggestion
  ifeq ($(call isTargetOs, macosx aix)+$(call isTargetOsEnv, bsd.netbsd), false+false)
```

Reply:
In fact, it can probably even better be written as:
```
ifeq ($(call isTargetOsType, unix)+$(call isTargetOs, macosx aix)+$(call isTargetOsEnv, bsd.netbsd), true+false+false)
```
and the comment changed to say something like "this is not needed on those unix variants where we have SCTP".

Reply:
Thanks, noted

Reply:
In fact, it can probably even better be written as:
```
ifeq ($(call isTargetOsType, unix)+$(call isTargetOs, macosx aix)+$(call isTargetOsEnv, bsd.netbsd), true+false+false)
```
and the comment changed to say something like "this is not needed on those unix variants where we have SCTP".

(I wrote this comment several days ago, but just now found an open tab with it un-posted. I apologize if I've managed to post this twice.)